packages/use-insertion-effect-with-fallbacks/src/index.js

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react'
+
+const isBrowser = typeof document !== 'undefined'
+
+const syncFallback = create => create()
+
+const useInsertionEffect = React['useInsertion' + 'Effect']
+  ? React['useInsertion' + 'Effect']
+  : false
+
+export const useInsertionEffectAlwaysWithSyncFallback = !isBrowser
+  ? syncFallback
+  : useInsertionEffect || syncFallback
```

Comment:
Please be extra careful when reviewing this as the shape of this has slightly changed when I moved this. I think that the current code is functionally equivalent to the old version though.